src/app/js/fields/index.js

```diff
@@ -138,13 +164,18 @@ const getDefaultField = (name, index, rest = {}) => ({
 export default handleActions(
     {
         ADD_FIELD: (state, { payload }) => {
-            const { name, ...rest } = payload || {};
+            const { name, subresource, ...rest } = payload || {};
```

Comment:
Avoid passing ...rest like arguments through, instead pick the variables you need from it.
Passing it fully makes it less maintainable and robust, if the payload changes because of some side effect, like adding an _id property, it would probably break mongoose.